sdk/src/main/java/io/verloop/sdk/Constants.kt

```diff
@@ -0,0 +1,11 @@
+package io.verloop.sdk
+
+object Constants {
+    internal const val JS_CALL_SET_USER_PARAMS = "JS_CALL: SetUserParams"
+    internal const val JS_CALL_SET_USER_ID = "JS_CALL: SetUserId"
+    internal const val JS_CALL_SET_DEPARTMENT = "JS_CALL: SetDepartment"
+    internal const val JS_CALL_SET_RECIPE = "JS_CALL: SetRecipe"
+    internal const val JS_CALL_SET_CUSTOM_FIELD = "JS_CALL: SetCustomField"
+    internal const val JS_CALL_SET_WIDGET_OPENED = " JS_CALL:WidgetOpened"
```

Comment:
Sometimes there is a space after the `:` and other time we are missing it. Let's keep it consistent

Reply:
Fixed